apps/next-evans/components/truth-defended/TruthDefendedNotes.tsx

```diff
@@ -0,0 +1,314 @@
+    <Footnote number={177}>Hebrews 9:9</Footnote>
+    <Footnote number={178}>
+      Matthew 5:21-22; 5:27-28, etc. 2 Corinthians 10:5, James 1:14-15
+    </Footnote>
+    <Footnote number={179}>Matthew 3:11-12</Footnote>
+    <Footnote number={180}>1 John 3:8</Footnote>
+    <Footnote number={181}>1 Peter 3:21</Footnote>
+    <Footnote number={182}>Ephesians 4:5, Romans 6:3,6-8, 11</Footnote>
+    <Footnote number={183}>Zechariah 13:1; John 13:8</Footnote>
+    <Footnote number={184}>1 Corinthians 6:11</Footnote>
+    <Footnote number={185}>John 3:30</Footnote>
+    <Footnote number={186}>
+      i.e. temple ceremonies, washings, circumcision, purification rites, physical
+      separation from Gentles, anointing with oil, avoiding blood and meat that had been
+      strangled, etc.
+    </Footnote>
+    <Footnote number={187}>Romans 14:17</Footnote>
+    <Footnote number={188}>1 Corinthians 4:20</Footnote>
+    <Footnote number={189}>
+      Galatians 5:2; Colossians 2:16-17, 22; Hebrews 9:9-10
+    </Footnote>
+    <Footnote number={190}>Acts 16:3; 21:20-28</Footnote>
+    <Footnote number={191}>1 Corinthians 9:20</Footnote>
+    <Footnote number={192}>1 Corinthians 1:14</Footnote>
+    <Footnote number={193}>1 Corinthians 1:17</Footnote>
+    <Footnote number={194}>Acts 26:18; Colossians 1:13</Footnote>
+    <Footnote number={195}>John 6:51,56</Footnote>
+    <Footnote number={196}>John 14:20,23</Footnote>
+    <Footnote number={197}>1 Peter 2:2-3</Footnote>
+    <Footnote number={198}>2 Peter 3:18</Footnote>
+    <Footnote number={199}>Revelation 3:20</Footnote>
+    <Footnote number={200}>Ephesians 5:30</Footnote>
+    <Footnote number={201}>John 15:5</Footnote>
+    <Footnote number={202}>Exodus 16:21</Footnote>
+    <Footnote number={203}>John 4:14</Footnote>
+    <Footnote number={204}>Acts 17:28</Footnote>
+    <Footnote number={205}>
+      John 13:4-15. Which outward washing could also have been regarded as a lasting
+      outward ordinance; for the words of Christ were: “If I then, your Lord and Teacher,
+      have washed your feet, you also ought to wash one another’s feet. For I have given
+      you an example, that you should do as I have done to you.”
+    </Footnote>
+    <Footnote number={206}>Acts 15:20,29</Footnote>
+    <Footnote number={207}>James 5:14-15</Footnote>
+    <Footnote number={208}>Galatians 5:19-24</Footnote>
+    <Footnote number={209}>Ephesians 5:9</Footnote>
+    <Footnote number={210}>1 John 3:10</Footnote>
+    <Footnote number={211}>Romans 8:14-9</Footnote>
+    <Footnote number={212}>Romans 12:9</Footnote>
+  </div>
+);
+
+export default TruthDefendedNotes;
+
+const Footnote: React.FC<{ number: number; children: React.ReactNode }> = ({
+  number,
+  children,
+}) => (
+  <div className="relative flex items-start">
+    <span
+      id={`note-${number}`}
+      className="absolute"
+      style={{
+        height: `70px`,
+        top: `calc(0px - 70px - 0.5em)`,
```

Comment:
here, and in one other place, i think we have a constant for this magic number top nav height (in the new next app, not just the old one). let's use that constant, having these magic numbers scattered makes me nervous.